src/Sylius/Bundle/ReportBundle/Controller/ReportController.php

```diff
@@ -67,6 +69,9 @@ public function embeddAction(Request $request, $report, array $configuration = a
         $configuration = $request->query->get('configuration', $configuration);
         $data = $this->get('sylius.report.data_fetcher')->fetch($report, $configuration);
 
-        return $this->get('sylius.report.renderer')->render($report, $data);
+        $response = new Response();
+        $response->setContent($this->get('sylius.report.renderer')->render($report, $data));
+
+        return $response;
```

Comment:
Wanted to keep the behaviour close to the [framework's EngineInterface](https://github.com/symfony/TwigBundle/blob/master/TwigEngine.php#L75) but perhaps it's not important.